crates/net/p2p/src/error.rs

```diff
@@ -193,9 +191,9 @@ pub enum DownloadError {
     #[display("received empty response")]
     EmptyResponse,
     /// Error while executing the request.
-    RequestError(RequestError),
+    RequestError(#[error(source)] RequestError),
```

Comment:
same, does it need the `error(source)` attribute when it's a 1-ary tuple struct?

Reply:
you're right, https://docs.rs/derive_more/latest/derive_more/derive.Error.html

sorry @tcoratger I was wrong earlier, if this is a tuple, then we don't need it

Reply:
fixed